lib/logger/src/styles.ts

```diff
@@ -16,9 +21,22 @@ export const styles = {
   dim: colours.grey,
   title: colours.bold.underline,
   taskHeader: colours.bgWhite.black,
-  errorHighlight: colours.red,
-  error: (string: string): string => `${styles.errorHighlight.bold('‼︎')} ${styles.title(string)}`,
-  warningHighlight: colours.yellow,
-  warning: (string: string): string => styles.warningHighlight.bold('⚠︎') + ' ' + string,
-  ruler: (): string => styles.dim('─'.repeat(process.stdout.columns / 2))
+  errorHighlight: colours.bgRed.bold,
+  error: (string: string): string => `${styles.errorHighlight(' × ️')} ${styles.title(string)}`,
```

Comment:
**todo**: there is a [cursed character](https://stackoverflow.com/questions/38100329/what-does-u-ufe0f-in-an-emoji-mean-is-it-the-same-if-i-delete-it) hidden after the multiplication sign which seems to confuse most text renderers (you can see the monospacing is broken in the diff on GitHub though it absolutely destroys my combo of tmux + nvim) probably meant to denote that the multiplication should be rendered as emoji but something seems to have gone wrong and the two code points have been split by a space.

you could either remove the code point
```suggestion
  error: (string: string): string => `${styles.errorHighlight(' × ')} ${styles.title(string)}`,
```

or fix the split between the two code points (though i'm not sure if the selector will have any effect on the multiplication character, i don't see it listed in this [table](http://unicode.org/emoji/charts/emoji-variants.html))
```suggestion
  error: (string: string): string => `${styles.errorHighlight(' ×️ ')} ${styles.title(string)}`,
```

or maybe use a different emoji 😄
```suggestion
  error: (string: string): string => `${styles.errorHighlight(' ✖️ ')} ${styles.title(string)}`,
```

Reply:
gahhhh. that's probably because i was using `‼︎` at one point which needs the variant selector to not render as 
‼️. `×` isn't a character with variants.